.github/workflows/review.yml

```diff
@@ -0,0 +1,45 @@
+name: Review
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+  workflow_dispatch:
+
+env:
+  GH_TOKEN: ${{ github.token }}
+  PR_NUMBER: ${{ github.event.number }}
+  REPO: ${{ github.repository }}
+
+permissions:
+  pull-requests: write
+
+jobs:
+  review-check:
+    runs-on: ubuntu-latest
+    outputs:
+      review-result: ${{ steps.review-check-result.outputs.script_result }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: 'main'
+    - name: Install Foundry
+      uses: foundry-rs/foundry-toolchain@v1
+    - name: Check PR
+      id: review-check-result
+      run: |
+        script_result=$(bash bin/github-review.sh ${PR_NUMBER} | tail -1)
+        echo "script_result=$script_result" >> $GITHUB_OUTPUT;
+
+  review-result:
+    runs-on: ubuntu-latest
+    needs: [review-check]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Post Success Comment to the PR
+      if: needs.review-check.outputs.review-result == 'Network addresses & Code hashes are correct'
+      run: |
+        gh pr comment "$PR_NUMBER" --repo "$REPO" --body "✅ Success: Network addresses & Code hashes are correct"
```

Comment:
Nit: Don't post this in a PR comment - just make CI fail (it will show as a failed check).

Reply:
My reasoning: if the dev is subscribed to this repo, he will get a message with this content, more specifically the "✅", easier to spot and faster to approve. Check success won't provide this way.

Reply:
My thought: if the dev is subscribed to this repo, he will get a message with this content, more specifically the "✅", easier to spot and faster to approve. Check success won't provide this way.

Reply:
But it will show as a red X here:

![image](https://github.com/safe-global/safe-deployments/assets/4210206/bc698830-f3f8-4eed-85b6-72f7f85de32f)

To me, this is super obvious.

Reply:
Updated 👍🏾